src/frontend/src/components/ChangeRequestDropdown.tsx

```diff
@@ -0,0 +1,75 @@
+import { MenuItem, Select, SelectChangeEvent } from '@mui/material';
+import { isWithinInterval, subDays } from 'date-fns';
+import { Control, Controller } from 'react-hook-form';
+import { wbsPipe } from 'shared';
+import { useAllChangeRequests } from '../hooks/change-requests.hooks';
+import LoadingIndicator from './LoadingIndicator';
+import { useCurrentUser } from '../hooks/users.hooks';
+
+interface ChangeRequestDropdownProps {
+  control: Control<any, any>;
+  name: string;
+}
+
+const ChangeRequestDropdown = ({ control, name }: ChangeRequestDropdownProps) => {
+  const user = useCurrentUser();
+  const { isLoading, data: changeRequests } = useAllChangeRequests();
+  if (isLoading || !changeRequests) return <LoadingIndicator />;
+
+  const today = new Date();
+  const fiveDaysAgo = subDays(today, 5);
+
+  const filteredRequests = changeRequests?.filter(
+    (cr) => !!cr.dateReviewed && isWithinInterval(cr.dateReviewed, { start: fiveDaysAgo, end: today })
```

Comment:
Great job with filtering here.

I would suggest removing !! in front of cr.dateReviewed and leaving it as just cr.dateReviewed since it functions the same and is easier to understand.

Also, from the issue description, it said to display all change requests that were **approved** in the past 5 days, so I would suggest we can add a condition of checking cr.accepted here.